web/src/portico/help.ts

```diff
@@ -184,5 +200,4 @@ window.addEventListener("popstate", () => {
 });
 
 $("body").addClass("noscroll");
-
```

Comment:
This also looks like an unintentional formatting change.

Reply:
fixed!